lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1215,24 +1216,30 @@ def print_redis_state(self):
             mme_ueip_imsi_map_entries,
         )
 
-    def enable_nat(self):
+    def enable_nat(self, ip_version=4):
         """Enable Nat"""
         self._set_agw_nat(True)
-        self._validate_nated_datapath()
+        self._validate_nated_datapath(ip_version)
         self.exec_command("sudo ip route del default via 192.168.129.42")
         self.exec_command("sudo ip route add default via 10.0.2.2 dev eth0")
 
-    def disable_nat(self):
+    def disable_nat(self, ip_version=4):
         """Disable Nat"""
-        self.exec_command("sudo ip route del default via 10.0.2.2 dev eth0")
-        self.exec_command(
-            "sudo ip addr replace 192.168.129.1/24 dev uplink_br0",
-        )
-        self.exec_command(
-            "sudo ip route add default via 192.168.129.42 dev uplink_br0",
-        )
+        if ip_version == 4:
+          self.exec_command("sudo ip route del default via 10.0.2.2 dev eth0")
+          self.exec_command(
+              "sudo ip addr replace 192.168.129.1/24 dev uplink_br0",
+          )
+          self.exec_command(
+              "sudo ip route add default via 192.168.129.42 dev uplink_br0",
+          )
+        else :
+          self.exec_command("sudo ip route del default via  2020::10 dev eth0")
+          self.exec_command("sudo ip addr replace 3001::10 dev uplink_br0")
+          self.exec_command("sudo ip route -A inet6 add default via 3001::2 dev uplink_br0")
```

Comment:
This is starting to get really complicated, Can you also add documentation in this file that lists all IPs and its VM and interface names.

Reply:
done